Matomo/ruleset.xml

```diff
@@ -47,4 +47,21 @@
         <!-- still allow those functions in tests -->
         <exclude-pattern>*/tests/*</exclude-pattern>
     </rule>
+
+    <rule ref="Matomo.Security.SensitiveParameter">
+        <properties>
+            <property name="sensitiveKeywords" type="array">
+                <element value="password"/>
+                <element value="passwordHash"/>
+                <element value="passwordHashed"/>
+                <element value="salt"/>
+                <element value="authToken"/>
+                <element value="tokenAuth"/>
+                <element value="token_auth"/>
+                <element value="secret"/>
+                <element value="apiKey"/>
```

Comment:
@sgiehl Can we add `access_token` this is used in AdvertisingConversionExport generally this will be masked, but I think this is the only sensitive keyword we use in plugins
```suggestion
                <element value="apiKey"/>
                <element value="access_token"/>
                 <element value="accessToken"/>
```